datafusion/core/tests/sql/timestamp.rs

```diff
@@ -1065,3 +1065,23 @@ async fn cast_to_timestamp_micros_twice() -> Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn to_timestamp_i32() -> Result<()> {
+    let ctx = SessionContext::new();
+
+    let sql = "select to_timestamp(cast (1 as int));";
```

Comment:
Can we please add `to_timestamp_seconds(..)`, `to_timestamp_milliseconds(..)` etc here?